### PR TITLE
feat(cleanup): Cleanup of a release

### LINKED ISSUE #122: support cleanup targeting one specific release
support cleanup targeting one specific release, right now it appears its all or nothing?

### PR DESCRIPTION
**Pull request by hickeyma**
Add functionality to cleanup command where user can specify a specific release to remove.

Fixes #122

### CODE REVIEW

File: cmd/cleanup.go [hunk 1]
```diff
@@ -60,6 +63,7 @@ func newCleanupCmd(out io.Writer) *cobra.Command {
 	settings.AddFlags(flags)
 
 	flags.BoolVar(&configCleanup, "config-cleanup", false, "if set, configuration cleanup performed")
+	flags.StringVarP(&releaseName, "name", "n", "", "the release name. When it is specified, the named release and its versions will be removed only. Should not be used with other cleanup operations")
```
Comment (marckhouzam) [inline cmd/cleanup.go]: I think the short form -n will not work because it is already used for the short form of --namespace as a global flag to helm.

Comment (rimusz) [reply cmd/cleanup.go]: we just could use then `--name` only

**Comment by rimusz** [comment]
also makes sense to add https://github.com/helm/helm-2to3/issues/122#issuecomment-597875154

File: cmd/cleanup.go (hunk 1 above)
Comment (hickeyma) [reply cmd/cleanup.go]: Updated removing the short form `-n`.

**Comment by hickeyma** [comment]
> also makes sense to add #122 (comment) print the release name to be deleted

Done.

**Comment by hickeyma** [comment]
Thanks for reviews @marckhouzam @boarder981 @rimusz for review and comments. Updated and ready for review again.

**Comment by marckhouzam** [comment]
Woops, @hickeyma the `completion.yaml` file should be updated to include the new `--name` flag.

**Comment by hickeyma** [comment]
> the completion.yaml file should be updated to include the new --name flag.

Good catch @marckhouzam. Updated and ready for review again.

**Pull request merged**